bears/vcs/git/CISkipInspectBear.py

```diff
@@ -0,0 +1,74 @@
+import re
+
+from bears.vcs.git.GitCommitMetadataBear import GitCommitMetadataBear
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+
+
+class CISkipInspectBear(GlobalBear):
+    LANGUAGES = {'Git'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    CI_PROVIDERS_METADATA = {
+          'CircleCI': r'\[ci skip\]|\[skip ci\]',
+          'TravisCI': r'\[ci skip\]|\[skip ci\]',
+          'AppveyorCI': r'\[skip ci\]|\[ci skip\]|\[skip appveyor\]',
+          }
+
+    BEAR_DEPS = {GitCommitMetadataBear}
+
+    def run(self, dependency_results,
+            ci_engines_used: list = [],
+            filename_globs: list = [],
+            **kwargs):
+        """
+        Inspect the HEAD commit to check if it disables
+        CI build.
+
+        :param ci_engines_used:     CI engines used in
+                                    the project.
+        :param filename_globs:      To identify files
+                                    that should not skip
+                                    CI builds.
+        """
+        for result in dependency_results[GitCommitMetadataBear.name]:
+
+            for ci_engine in ci_engines_used:
```

Comment:
if the user doesnt supply a value for `ci_engines_used`, this will do nothing.

a bear which does checks shouldnt do *nothing* when running without any explicit setting overrides in .coafile.